addons/test_mass_mailing/tests/test_blacklist.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests.common import users
+from odoo.addons.test_mass_mailing.tests import common
+from odoo.exceptions import AccessError
+
+
+class TestBLAccessRights(common.MassMailingCase):
+
+    @users('emp')
+    def test_bl_create_employee(self):
+        with self.assertRaises(AccessError):
+            self.env['mail.blacklist'].create([{'email': 'Arya.Stark@example.com'}])
+
+    @users('marketing')
+    def test_bl_create_marketing(self):
+        self.env['mail.blacklist'].create([{'email': 'Arya.Stark@example.com'}])
+
```

Comment:
Migrate existing test for access rights from mass_mailing test_blacklist_access_rights file into this